docs/content/docs/concepts/signals.md

```diff
@@ -6,6 +6,95 @@ discussion: 5469
 aliases:
   - "/docs/advanced/signals/"
   - "/docs/deep/signals/"
+draft: true
 ---
 
-Signals are similar to ops, but are not stored; they're useful to implement features like user presence.
+Signals are similar to ops, but are not stored; they're useful to implement features like user presence. They provide a simple way to communicate transient data to connected clients within a Fluid application.
+
+They are lossy and not guaranteed to be ordered on delivery relative to other signals and ops. A good analogy to help conceptualize signals is that they can be interpreted as UDP protocol, whereas standard ops can be thought of as TCP protocol.
```

Comment:
Cool, I will definitely try to reach out! I put this since this is how signals were described by Skyler in my project description. He also included in the [ADO Task](https://dev.azure.com/fluidframework/internal/_workitems/edit/1146) to let the customers know that they are not persisted nor guaranteed. But I will definitely try to get more clarity around what exactly is guaranteed and what isn't when a signal is submitted.

Reply:
I also looked into the IFluidDataStoreRuntime to see it's signal guarantees here: [@fluidframework/datastore-definitions](https://github.com/microsoft/FluidFramework/tree/main/packages/runtime/datastore-definitions). 

> Signals are not persisted, ordered, or guaranteed. If a client is behind, the op state can be behind the signal state. For this reason people usually stick the currentSeq on the signal, so other clients can wait to process if they are behind.